src/assets/scripts/client/game/ScoreLogic.js

```diff
@@ -0,0 +1,139 @@
+import EventBus from '../lib/EventBus';
+import GameController, { GAME_EVENTS } from './GameController';
+import UiController from '../ui/UiController';
+import { AIRCRAFT_EVENT } from '../constants/eventNames';
+import { nm_ft } from '../utilities/unitConverters';
+
+/**
+ * @class ScoreLogic
+ */
+export default class ScoreLogic {
```

Comment:
Not the best name. If someone has a better idea, let me know.

Reply:
`PlayerScoreController` would be closer to the conventions we're currently using

Reply:
renamed to ScoreController. Not a fan of the term `player` as it suggests openScope is a game